GetandTake/Pages/Products/Create.cshtml.cs

```diff
@@ -0,0 +1,52 @@
+using GetandTake.Models;
+using GetandTake.Models.DTOs.BaseDTO;
+using GetandTake.Services.Abstract;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace GetandTake.Pages.Products;
+
+public class CreateModel : PageModel
+{
+    private readonly IProductService _productService;
+    private readonly ICategoryService _categoryService;
+    private readonly ISupplierService _supplierService;
+
+    public IEnumerable<Category> Categories { get; private set; }
+
+    public IEnumerable<Supplier> Suppliers { get; private set; }
+
+    [BindProperty]
+    public ProductDTO ProductDTO { get; set; }
+
+    public CreateModel
+        (
```

Comment:
No need to wrap these brackets to separate row, let's try to stick with the following template:
```suggestion
        public CreateModel(
             IProductService productService,
             ICategoryService categoryService,
             ISupplierService supplierService)
```